### PR TITLE
Add email documentation

### PR DESCRIPTION
**Pull request by timlegge**
Add email documentation - need to check whether the smtp support was merges in kimai

### CODE REVIEW

File: documentation/email.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Email
+
+Currently kimai only uses email to send users an email if they have forgotten their password.  It is also currently only implemented for the kimai_auth authenticator.  This is reasonable as the other authentication methods do not allow kimai to change passwords (an LDAP user must goto to the LDAP administrator if they forget their password.
```
Comment (kevinpapst) [inline documentation/email.md]: "change passwords (an LDAP" => "change passwords, e.g. an LDAP"

Comment (kevinpapst) [reply documentation/email.md]: kimai => Kimai

File: documentation/email.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## Configuration
+
+Currently configuring email requires the use of the ``includes/autoconf.php`` file and the configuration is performed on installation.  The values can be changed there later if necessary.
```
Comment (kevinpapst) [inline documentation/email.md]: "performed on installation" => "performed during installation"

File: documentation/email.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+```
+
+Set the parameters according to the email documentation below.
+
+## Email Configuration
+
+### Configuration-parameters
+
+* **mail_transport:** This defines the Zend Mail transport mechanism used.  Default is **sendmail** (requires no further configuration).  **smtp** requires the following variables to be set as required by your smtp server
+* **smtp_name:** The DNS server name of your smtp server
```
Comment (kevinpapst) [inline documentation/email.md]: Really the DNS server?

File: documentation/email.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+* **smtp_host:** The DNS or IP address or your SMTP Server
```
Comment (kevinpapst) [inline documentation/email.md]: DNS => domain or host name?

**Comment by timlegge** [comment]
Recommended changes have been made

**Comment by kevinpapst** [comment]
Thank you @timlegge

**Pull request merged**